.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,4 +19,5 @@ This pull request still needs...
 
 ### Formatting
 
-- [ ] Ran `make formatall`.
+- [ ] Ran `make format`.
+- [ ] Fixed errors surfaced by `make lints`.
```

Comment:
Why not tell them to run `make clippy` instead since they're already running `format`? Then you could get rid of `lints` altogether and just have `ci-lints` call `check-format` and `clippy` separately.

Reply:
Or put a different way, do we really need a single target for running both operations?